tests/e2e/network/service_annotations.go

```diff
@@ -315,3 +347,33 @@ func validateInternalLoadBalancer(c clientset.Interface, ns string, url string)
 	utils.Logf("validation finished")
 	return err
 }
+func validateLoadBalancerBackendPools(vmssname string ) {
+	By("Waiting for service exposure")
+				pip, err := utils.WaitServiceExposure(cs, ns.Name, serviceName)
```

Comment:
please run gofmt -s -w for this file

Reply:
error: cannot use -w with standard input

Reply:
ok done